packages/content/src/lib/marked-setup.service.ts

```diff
@@ -11,11 +11,13 @@ import 'prismjs';
 import 'prismjs/plugins/toolbar/prism-toolbar';
 import 'prismjs/components/prism-bash';
 import 'prismjs/components/prism-css';
+import 'prismjs/components/prism-diff';
```

Comment:
I don't know how much these increase the bundle size, but can these be added in the project instead of adding it here? Example

```ts
import { ApplicationConfig } from '@angular/core';
import { provideHttpClient } from '@angular/common/http';
import { provideClientHydration } from '@angular/platform-browser';
import { provideFileRouter } from '@analogjs/router';
import { provideContent, withMarkdownRenderer } from '@analogjs/content';
import 'prismjs/components/prism-diff';
import 'prismjs/plugins/diff-highlight/prism-diff-highlight';

export const appConfig: ApplicationConfig = {
  providers: [
    provideFileRouter(),
    provideClientHydration(),
    provideHttpClient(),
  ],
};
```

That way, if you don't use it, it's not always added to the bundle. The renderer code would remain though. Thoughts?

Reply:
Sounds good, will remove here, add some checks to make sure they're imported, and add docs showing how to add the imports. Thanks!